### DISCUSSION TITLE
screens

### DISCUSSION THREAD

**Discussion by McGee122** [flybywiresim/aircraft#2426] [Help] [answered]
in A320nx ı can't open the screens can anyone help me???

**Comment by wpine215** [comment]
Can you please be more elaborate as to what you mean by screens? Can you provide screenshots and write a couple more sentences on the issue?

**Reply by McGee122** [reply]
<img width="1913" alt="Ekran görüntüsü 2020-12-14 152457" src="https://user-images.githubusercontent.com/75931636/102081034-97b48d80-3e20-11eb-913b-92a672f54384.png">
like this ı re upload the msfs2020 but nothing changed

**Reply by hotshotp** [reply]
please close your sim, delete your a32nx folder and redownload it via the installer

**Reply by metronorth** [reply]
I've got a similar issue. Posted my own thread just now. Thanks for sharing. I'll attempt what hotshotp advised and will let you all know if that works.

**Comment by DarkOfNova** [comment, marked as answer]
I'll assume this worked for you and it was just a faulty installation.
